code/world.dm

```diff
@@ -140,13 +140,12 @@ var/savefile/panicfile
 
 	src.update_status()
 
-	sleep_offline = 1
```

Comment:
Just had a thought: should I only set this to 0 if we're running unit tests?

Reply:
🤔

Reply:
Probably, but it is unlikely to ever have a functional effect

Reply:
I'll just leave this as is, then